src/import/ImportAUP.cpp

```diff
@@ -1410,8 +1360,6 @@ bool AUPImportFileHandle::HandleImport(XMLTagHandler *&handler)
 
    // Handle other attributes, now that we have the tracks.
    // Apply them to all new wave tracks.
-   ++mAttrs;
```

Comment:
I think you need a pop-front of mAttrs to preserve this old behavior now, skipping "filename" before examining the rest.

Did you test importing of an Audacity 2 file?

Reply:
Not this specific case. wavetrack doesn't handle "filename" attribute anyway, so it should be safe. I will add pop_front anyway.